src/providers/ldap/sdap_async.c

```diff
@@ -642,6 +643,12 @@ struct tevent_req *sdap_exop_modify_passwd_send(TALLOC_CTX *memctx,
     DEBUG(SSSDBG_TRACE_INTERNAL,
           "ldap_extended_operation sent, msgid = %d\n", msgid);
 
+    stat_info = talloc_asprintf(state, "server: modify passwd dn: [%s]",
+                                sdap_get_server_ip_str(state->sh), user_dn);
+    if (stat_info == NULL) {
+        DEBUG(SSSDBG_OP_FAILURE, "Failed to create info string, ignored.\n");
+    }
+
     ret = sdap_op_add(state, ev, state->sh, msgid, NULL,
```

Comment:
Shouldn't NULL be replaced by `stat_info`?

Reply:
fixed